cg/constants/observations.py

```diff
@@ -78,7 +77,4 @@ class BalsamicObservationsAnalysisTag(StrEnum):
 class BalsamicLoadParameters(Enum):
     """Cancer Loqusdb load command parameters."""
 
-    PROFILE_THRESHOLD: float = 0.95
-    GQ_THRESHOLD: int = 10
-    HARD_THRESHOLD: float = 0.95
-    SOFT_THRESHOLD: float = 0.90
+    QUAL_THRESHOLD: int = 10
```

Comment:
Consider not using abbreviations, matter of taste though but I think it is usually better to be explicit. Feel free to disregard 😅

Reply:
Sometimes, using abbreviations increases the cognitive load when reading the code

Reply:
I agree, but we're safe here since this refers to a VCF format fixed field, `QUAL`.

Thanks for looking into this 💟